test/specs/new/nested_square_link.md

```diff
@@ -1,3 +1,3 @@
 [the `]` character](/url)
 
-[the ` character](/url)
+[the \` character](/url)
```

Comment:
So this PR is going to introduce a regression but fix a redos issue?

Reply:
yes